src/main/java/org/gridsuite/study/server/StudyException.java

```diff
@@ -73,6 +73,7 @@ public enum Type {
         VOLTAGE_INIT_PARAMETERS_NOT_FOUND,
         CREATE_VOLTAGE_INIT_PARAMETERS_FAILED,
         UPDATE_VOLTAGE_INIT_PARAMETERS_FAILED,
+        SUPERVSION_ERROR_DELETE_RESULTS,
```

Comment:
for homogeneous semantic DELETE_RESULTS_FAILED is better I think.
Also, if it's only used by supervision scope, maybe we should create another class of Exception

Reply:
Done